### ISSUE TITLE
Build cobol-check as GraalVM native-image

### ISSUE THREAD

**Issue by 0xE282B0** [openmainframeproject/cobol-check#220]
Hi,
I was testing cobol-check for the gnucobol language track of the [exercism.org](https://exercism.org/) learning platform. 
I'm concerned that installing a Java program would cause support requests to set up a local environment. Therefore I'd suggest building cobol-check as GraalVM native-image.

I already tried it successfully [with this branch](https://github.com/0xE282B0/cobol-check/tree/feature/graal-native-image), is this a contribution you would be interested in?

**Comment by dakaa16** [comment]
Hi @0xE282B0 

This would be a really fine addition - thank you for sharing this. I'd like to know how to run the exe-file, as I simply get: 
`Error: No bin\java.exe and no environment variable JAVA_HOME`

**Comment by 0xE282B0** [comment]
You're right, I forgot to include the bundle files and it tried to fall back to a Java VM.
Can you try it again?

**Comment by dakaa16** [comment]
This got me further. I get an exception when the executable is trying to get some copybooks that should be included as a resource. Messages, which are included in the same way works fine. The copybooks are placed at 
 `src/main/resources/org/openmainframeproject/cobolcheck/copybooks`
Is this something you can fix? Let me know if you need further information.
